Gateway/crds-angular/Services/GroupService.cs

```
@@ -273,6 +284,57 @@ public List<GroupDTO> GetGroupsByTypeForParticipant(string token, int participan
             var groupDetail = groupsByType.Select(Mapper.Map<Group, GroupDTO>).ToList();
             return groupDetail;
         }
+        
+        public Participant GetParticipantRecord(string token) 
+        {
+            var participant = _participantService.GetParticipantRecord(token);
+            return participant;
+        }
+
+        public int SendJourneyEmailInvite(EmailCommunicationDTO communication, string token)
+        {
+            var participant = GetParticipantRecord(token);
+            var groups = GetGroupsByTypeForParticipant(token, participant.ParticipantId, 19);
+            var membership = groups.Where(group => group.GroupId == communication.groupId).ToList();
+            if (membership.Count <= 0)
+            {
+                //notfound, don't send invite
+                return 1;
```

Would it be better to throw an exception here instead of returning an int?